iphone/Classes/TiUITableView.m

```diff
@@ -454,7 +454,9 @@ - (UITableView *)tableView
 
     defaultSeparatorInsets = [tableview separatorInset];
 
-    tableview.layoutMargins = UIEdgeInsetsZero;
+    if (style != UITableViewStyleInsetGrouped) {
+      tableview.layoutMargins = UIEdgeInsetsZero;
+    }
```

Comment:
Pretty sure this crashes on Xcode 10 and iOS < 13. Maybe:
```objc
#if IS_XCODE_9
if ([TiUtils isiOSVersionOrGreater:@"13.0"] && style != UITableViewStyleInsetGrouped) {
	tableview.layoutMargins = UIEdgeInsetsZero;
}
#else
tableview.layoutMargins = UIEdgeInsetsZero;
#endif
```

Reply:
You are right. Fixed. Thanks!